### ISSUE TITLE
是开源的吗？

### ISSUE THREAD

**Issue by cheaprYang** [wangweianger/zanePerfor#1]
我想部署一套测试一下

**Comment by wangweianger** [comment]
@cheaprYang
是的 我今天会把部署文档写完，请关注 https://blog.seosiwei.com/performance/index.html 开发文档

**Comment by giscafer** [comment]
@wangweianger 昨天本地单机部署试了一下，创建应用也是拿本机跑得前端，地址写localhost的，用通用上报插件没有效果。看不到pv，也上报不了error。

自测后端api get请求是通的，有日记。但是使用通用上报sdk就不行了。（npm 安装的performance-report）, 无任何效果，也无任何错误。用法和npm 模块文档一致

**Comment by Lanseria** [comment]
里面有文字错误不是`.../webscript` 而是`.../web`
